app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryScreenModel.kt

```diff
@@ -267,7 +268,7 @@ class LibraryScreenModel(
 
         fun LibrarySort.comparator(): Comparator<LibraryItem> = Comparator { i1, i2 ->
             when (this.type) {
-                LibrarySort.Type.Alphabetical -> {
+                LibrarySort.Type.Alphabetical, LibrarySort.Type.Random -> {
```

Comment:
As fallback so this function doesn't have to be nullable. This is bypassed by the map function in the random case.

Reply:
do something like 
```kt
LibrarySort.Type.Random -> {
    error("Should never throw")
}
```

Reply:
Much better, thanks